test/bundler/transpiler/zod/diff-fixture.ts

```
@@ -0,0 +1,510 @@
+    z.object({
+      req: z.string(),
+      opt: z.string().optional(),
+      def: z.string().default("d"),
+      nul: z.string().nullable(),
+      nish: z.number().nullish(),
+    }),
+  objAnyProp: () => z.object({ a: z.any() }),
+  objUndefinedProp: () => z.object({ u: z.undefined() }),
+  objLiteralUndefined: () => z.object({ u: z.literal(undefined) }),
+  // Duplicate literal keys: the object zod receives keeps the first position and the last value.
+  objDupKey: () => z.object({ a: z.string(), b: z.number(), a: z.boolean() } as any),
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)